pom.xml

```diff
@@ -237,6 +237,34 @@
         </plugins>
       </build>
     </profile>
+    <profile>
+      <id>jar-with-driver-and-dependencies</id>
+      <build>
+        <plugins>
+          <plugin>
+            <groupId>org.apache.maven.plugins</groupId>
+            <artifactId>maven-assembly-plugin</artifactId>
+            <version>2.4.1</version>
+            <configuration>
+              <!-- get all project dependencies -->
+              <descriptorRefs>
+                <descriptorRef>jar-with-dependencies</descriptorRef>
```

Comment:
nit: Should this be `jar-with-driver-and-dependencies`

Reply:
yeah, I tried that, but didn't work. Apparently this is not free-form text. Here's the error I got:
{code}
[ERROR] Failed to execute goal org.apache.maven.plugins:maven-assembly-plugin:2.4.1:single (make-assembly) on project jdbc-socket-factory-parent: Error reading assemblies: Descriptor with ID 'jar-with-driver-and-dependencies' not found -> [Help 1]
{code}

Reply:
Ah yup: https://maven.apache.org/plugins/maven-assembly-plugin/descriptor-refs.html#jar-with-dependencies

Ok, thanks